### ISSUE TITLE
new metric idea: ratio of current 7-day growth vs. previous 7-day growth

### ISSUE THREAD

**Issue by dmaymudes** [davidbau/covid-19-chart#55]
Now that total cumulative cases are a less-useful metric, could it make sense to try to estimate current spread by computing the ration between this week's growth and the previous week's?

**Comment by davidebbo** [comment]
Yep, I was thinking about that one too after seeing it used in some news coverage. I agree it would better capture the locations going through a 'relapse'.
